player.rb

```diff
@@ -0,0 +1,19 @@
+# class player
+class Player
+  attr_accessor :value, :symbol, :name
+  
+  PLAYER1_SYMBOL = 'X'
+  PLAYER1_NAME = 'Player 1'
+  PLAYER2_SYMBOL = 'O'
+  PLAYER2_NAME = 'Player 2'
```

Comment:
Why constants? These should be parameters

Reply:
The names are wrong, they are just default values in case the user doesn't type one. We renamed them for better understanding, for example, `PLAYER1_NAME` is now `PLAYER1_DEFAULT_NAME`.